pkg/kubectl/pdb_test.go

```diff
@@ -170,12 +183,14 @@ func TestPodDisruptionBudgetV2Generate(t *testing.T) {
 		t.Errorf("unexpected error: %v", err)
 	}
 
-	tests := map[string]struct {
+	tests := []struct {
```

Comment:
I like maps as much as I like arrays

Reply:
I can't agree that, no matter what else, the use of arrays in kubernetes is a bit more.